### PR TITLE
Foldable Wheelchairs (30 Minute Port Edition)

### PR DESCRIPTION
**Pull request by Skies-Of-Blue**
## About The Pull Request
Direct port of https://github.com/tgstation/tgstation/pull/53796, making non-motorized wheelchairs able to be folded. This was a cut-paste job with minor tweaking, if something breaks please yell at me.

## Why It's Good For The Game
Zevo can play their gay little lizard in peace

wowee this should have been in years ago

Also allows for medships to keep wheelchairs in stock or smth idk

## Changelog

:cl:
tweak: non-motorized wheelchairs can now be folded
imageadd: a few sprites to support the above
/:cl:

### CODE REVIEW

**Comment by tmtmtl30** [comment]
i feel like normal-sized is too small? it's a wheelchair, how are you fitting seven of them in a backpack...

**Comment by Zevotech** [comment]
Please add a do_after to it so that it isnt instant and make sure you can still do it while prone as a paraplegic

**Comment by Skies-Of-Blue** [comment]
Requested changes made!

File: code/modules/vehicles/wheelchair.dm [hunk 1]
```diff
@@ -108,8 +110,42 @@
...
+	base_icon_state = "wheelchair_folded"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	w_class = WEIGHT_CLASS_BULKY
+	force = 8 //Force is same as a chair
+	var/unfolded_type = /obj/vehicle/ridden/wheelchair
+
+/obj/vehicle/ridden/wheelchair/MouseDrop(over_object, src_location, over_location)  //Lets you collapse wheelchair
+	. = ..()
+	if(over_object != usr || !Adjacent(usr) || !foldabletype)
```
Comment (west3436) [inline code/modules/vehicles/wheelchair.dm]: `if(over_object != usr || !Adjacent(usr) || !foldabletype || !ishuman(usr) || has_buckled_mobs())`
Unless I'm missing something as to why you didn't just have these combined before

Comment (thgvr) [reply code/modules/vehicles/wheelchair.dm]: yeah fix that

**Pull request merged**